server/plugin/api.go

```diff
@@ -408,6 +408,11 @@ func (p *Plugin) completeConnectUserToGitHub(c *Context, w http.ResponseWriter,
 		return
 	}
 
+	// track the successful connection
+	if err := p.tracker.TrackUserEvent("account_connected", c.UserID, nil); err != nil {
+		c.Log.WithError(err).Warnf("Failed to track telemetry 'account_connected' event")
+	}
```

Comment:
Let's make use of https://github.com/mattermost/mattermost-plugin-github/pull/653
```suggestion
	// track the successful connection
	p.TrackUserEvent("account_connected", c.UserID, nil)